pkg/platform/abstract/platform.go

```diff
@@ -613,7 +613,8 @@ func (ap *Platform) isSDKLogLine(logLine []byte) bool {
 
 func (ap *Platform) shouldAddToBriefErrorsMessage(logLevel uint8, logMessage, workerID string) bool {
 	knownFailureSubstrings := [...]string{"Failed to connect to broker"}
-	ignoreFailureSubstrings := [...]string{string(common.UnexpectedTerminationChildProcess)}
+	ignoreFailureSubstrings := [...]string{string(common.UnexpectedTerminationChildProcess),
+		string(common.FailedReadFromConnection)}
```

Comment:
Irrelevant to the PR (another case of a message that shouldn't be in the brief errors message)